### PR TITLE
added some handy docker example aliases

### PR DESCRIPTION
**Pull request by natemarks**

### CODE REVIEW

File: bashrc.d/docker.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+# run a new container from an image: dr-run <image>
+dr-run() {
+  docker run -it $1
+}
+
+# run a new container from an image: dr-run <image>
+drf-run() {
+  imageId="$(docker images | fzf | awk {'print $1":"$2'})"
```
Comment (sertvitas) [inline bashrc.d/docker.sh:20]: Should the single-quotes go outside the curly braces for the awk print?

File: bashrc.d/docker.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+# syntax: dr-run-dead
+dr-run-dead() {
+  docker commit $1 $1-image > /dev/null
+  docker run -it $1-image$*
+}
+
+# syntax: dr-reset-log
+dr-reset-log() {
+  truncate -s 0 $(docker inspect --format='{{.LogPath}}' $1)
```
Comment (sertvitas) [inline bashrc.d/docker.sh:63]: Should this have double-quotes around the: 
"$(docker inspect --format='{{.LogPath}}' $1)"
SC2046

**Review by sertvitas — CHANGES_REQUESTED** [review]
I added comments for the shellcheck warnings on lines 20 and 63.

Warning: ./bashrc.d/docker.sh:20:40: warning: This { is literal. Check expression (missing ;/\n?) or quote it. [SC1083]
Warning: ./bashrc.d/docker.sh:20:56: warning: This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]

Warning: ./bashrc.d/docker.sh:63:17: warning: Quote this to prevent word splitting. [SC2046]

File: bashrc.d/docker.sh (hunk 2 above)
Comment (natemarks) [reply bashrc.d/docker.sh:63]: oh I broke my own rule and forgot to run shellcheck. fixing

File: bashrc.d/docker.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# exec a command in a running container: dr-cmd abcd1234 cmd...
+dr-exec() {
+  docker exec -it $1 "${@:2}"
+}
+
+# exec a command using fzf to target a running container: dr-cmd  cmd...
+drf-exec() {
+  containerId=$(docker ps | egrep -v 'CONTAINER ID' | fzf | cut -d" " -f1)
```
Comment (sertvitas) [inline bashrc.d/docker.sh:9]: This line complained about 'egrep' and suggested 'grep -E' instead

Notice: ./bashrc.d/docker.sh:9:29: note: egrep is non-standard and deprecated. Use grep -E instead. [SC2196]

**Review by sertvitas — APPROVED** [review]

**Pull request merged**